parlai/tasks/hotpotqa/build.py

```diff
@@ -9,9 +9,10 @@
 import os
 import json
 
-TRAIN_FILENAME = 'hotpot_train_v1.1.json'
-DEV_DISTRACTOR_FILENAME = 'hotpot_dev_distractor_v1.json'
-DEV_FULLWIKI_FILENAME = 'hotpot_dev_fullwiki_v1.json'
+VERSION = '1'
+TRAIN_FILENAME = f'hotpot_train_v{VERSION}.1.json'
```

Comment:
technically we support python 3.5 and so we shouldn't have f-strings.

we've been debating changing that...